filters/exclusions.txt

```diff
@@ -250,18 +250,167 @@ _campaign=$popup
+~mail.yandex.*,~vh.yandex.ru,~yandex.net,yandex.*#?#+js(no-setTimeout-if, t.apply(n, 25)
+~mail.yandex.*,~vh.yandex.ru,~yandex.net,yandex.*#?#+js(no-setTimeout-if, Ya.Rum.getSetting, 15)
+~vh.yandex.ru,~yandex.net,yandex.*#?#+js(abort-on-property-read, Ya.loaderVer)
+~yandex.net,yandex.*#?#+js(abort-on-property-read, yaads.renderBlock)
+~yandex.net,yandex.*#?#+js(abort-on-property-read, yaads.renderWidget)
+~yandex.net,yandex.*#?#+js(cookie-remover, tvoid)
+~yandex.net,yandex.*#?#+js(remove-attr, data-counter, .serp-list a)
+~yandex.net,yandex.*#?#+js(set-constant, __AAB__, false)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype._getAdConfig, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype._hasAdblock, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.getHoneypot, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.getYaads, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.initWidgetBlock, falseFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.isOpenImageUnauthPopupEnabled, false)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.loaderVer, noopFunc)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.los, null)
+~yandex.net,yandex.*#?#+js(set-constant, Object.prototype.useAbdBundle, false)
+~yandex.net,yandex.*#?#+js(set-constant, home.export.adb, {})
+!
+! Filter name: Official Polish filters for AdBlock, uBlock Origin & AdGuard
+~biblio.com,~milionczesci.pl,~dizibox.com,~2dehands.be,~ru,~ua,~www.pudelek.pl,~btdb.eu,~ewybory.eu ##[id^="adv-"]:not(#adv-settings)
+www.dobreprogramy.pl##+js(nano-remove-elements-onready.js, #phContent_avastBadge)
+filmweb.pl##+js(nano-remove-elements-onready.js, #skyBanner)
+$popunder,domain=rabatio.com|alerabat.com|salesandshopping.pl|kodyrabatowe.olx.pl|kod.rabatowy.pl
+polygamia.pl##+js(nano-remove-elements-onready.js, .screening)
+napiprojekt.pl##+js(insert-iframe, 1, honeypot, https://www.napiprojekt.pl/napisy-42731-Ostatni-U-Boot-(1993), width: 1px !important; height: 1px !important; position: absolute !important; left: -3000px !important)
+ppe.pl##+js(ac, active, #m_hot_1)
+ppe.pl##+js(ac, box_active, #hot_1)
+money.pl##div[class]:if(>div[class]:first-child:has-text(REKLAMA):if-not(>*))
+!
+! Filter name: Polish GDPR-Cookies Filters
+nike.com##body[data-overlay="cookieModalSimple"]:watch-attr(data-overlay):not([data-overlay="true"]) #gen-nav-commerce-header-v2 > div.pre-modal-window:has(> .pre-modal > .pre-modal-view > .pre-cookie-modal-body)
+!
+! Filter name: Polish Social Filters
+oszczedzarka.co.uk.textwidget > p > img[src="/img/ico/skype.png"]
+!
+! Filter name: road-block light
+science.hotnews.ro###\$\{tile\.name\}_\$\{tile\.id\}"
+||imasdk.googleapis.com/js/sdkloader/ima3.js$script,important,redirect=google-ima.js,domain=a1.ro|antena3.ro|antenastars.ro|as.ro|jurnalul.ro|observator.tv|observatornews.ro|spynews.ro|zutv.ro
```

Comment:
If I'm not wrong `redirect=google-ima.js` is used by uBlock Origin, so maybe we should add an alias for it.

Reply:
https://github.com/AdguardTeam/FiltersCompiler/issues/167